packages/core/chrome/core-chrome-browser-internal/src/ui/header/collapsible_nav.tsx

```diff
@@ -165,6 +165,14 @@ export function CollapsibleNav({
       {customNavLink && (
         <>
           <EuiFlexItem grow={false} style={{ flexShrink: 0 }}>
+            <EuiScreenReaderLive focusRegionOnTextChange>
+              {i18n.translate('core.ui.primaryNav.screenReaderAnnouncement', {
+                defaultMessage: 'Visit {title} application',
+                values: {
+                  title: appId,
+                },
+              })}
+            </EuiScreenReaderLive>
```

Comment:
Placing this **inside** the `EuiCollapsibleNav` isn't quite doing what you think it will - it's now located in the conditionally rendered flyout as opposed to living statically on the page, which is where live regions need to live.

So for this usage to make sense:

1. It likely needs to live in/next to the `button` JSX being passed to this component, not in the collapsible nav itself
2. Its text should not be imperative. We're not telling users to `Visit X application` - we're saying `You're now on X page after having performed a navigation action`. I would recommend **just** printing the current document.title/page title as that will get you pretty far, and it mimics standard non-SPA-navigation (the document title gets read out first on page load).